bedrock/settings/base.py

```diff
@@ -937,6 +937,10 @@ def lazy_email_backend():
 FXA_IFRAME_SRC = config('FXA_IFRAME_SRC',
                         default='https://accounts.firefox.com/')
 
+FXA_IFRAME_SRC_DIST_MAP = {
```

Comment:
For future updates/reference, could we add a short description of the use of this dict along with a bug number?